### ISSUE TITLE
I'd suggest you report/remove issues from Russian Nazi Bots

### ISSUE THREAD

**Issue by altrussia** [vshymanskyy/StandWithUkraine#214]
It seems that this repo is being used and possibly others as source for propaganda in favour of Russia. It doesn't seem to bother them to use this repo as a source even if it's obviously against Russian invasion.

Here's an example on r/russia

https://www.reddit.com/r/russia/comments/12dr57p/comment/jf7se6r/

Removing/Hiding issues doesn't let them use them to propagate it even further. 

Cheers!

**Comment by webknjaz** [comment]
Now, that's nice of reddit:
![Screenshot_2023-04-07-02-51-29-27_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://user-images.githubusercontent.com/578543/230518731-1265a07b-be6a-4f10-9ef7-8ed561a83b98.jpg)

**Comment by webknjaz** [comment]
@altrussia thanks for the report! I'm torn on this. On one hand, we don't do such cleanups, since it's useful to demonstrate how said people behave, on the other — a part of our group gathered around this resource have already suggested disabling the issues in the repo altogether. Personally, I mostly just lock things only when they are exceedingly toxic and outright spam.
Besides, if you're worried that they are used for propaganda, removing the issues won't stop that since they'll just use that too, inventing some fairytale about censorship 🤷‍♂️
Also, at least some of the issues are in the web archive, being ultimately linkable anyway...

**Comment by webknjaz** [comment]
> Here's an example on r/russia

Could you share a screenshot, by the way? reddit doesn't make it easy to preview..

**Comment by altrussia** [comment]
Here:

![image](https://user-images.githubusercontent.com/115747168/230616418-1e1d6bcc-af46-4029-b93b-40567439655f.png)

I honestly don't know how he got to your repo. But this event is very unusual.

**Comment by altrussia** [comment]
> Also, at least some of the issues are in the web archive, being ultimately linkable anyway...

Yes, hmm well. One thing I noticed while fighting relentlessly against disinformation is that anything can be used as disinformation even using this repo how ironically it might sound.

That said, I have good news. That user just cleaned up his own comment history of the last 2 months. It is also very unusual. I mean, I and other have been arguing with this persona for about 6 months and maybe more. But recently, I'd say his behaviour changed as if someone else was handling the account.

But taking a source from a repo on github named "StandWithUkraine" while he has been actively against Ukraine is strange to say the least.

And a day after I informed you about it, he decided to clean his recent history. It could be a coincidence or it could be someone covering his tracks after exposing a link that shouldn't be there at all. It's mind boggling honestly.

**Comment by webknjaz** [comment]
> Yes, hmm well. One thing I noticed while fighting relentlessly against disinformation is that anything can be used as disinformation even using this repo how ironically it might sound.

Good point. After giving it some thought, I've updated the issue to just state that there was spam in the post. The history is visible for the curious ones. I'll probably keep the issues but maybe add similar text there.

> That said, I have good news. That user just cleaned up his own comment history of the last 2 months. It is also very unusual. I mean, I and other have been arguing with this persona for about 6 months and maybe more. But recently, I'd say his behaviour changed as if someone else was handling the account.
> 
> But taking a source from a repo on github named "StandWithUkraine" while he has been actively against Ukraine is strange to say the least.
> 
> And a day after I informed you about it, he decided to clean his recent history. It could be a coincidence or it could be someone covering his tracks after exposing a link that shouldn't be there at all. It's mind boggling honestly.

Coincidence or routine, what you describe kinda matches the behavior of "bot farms" that the ruscists are known to employ in their cyber warfare / disinformation campaigns. So I'm personally not surprised much.
